modules/local/reformat_rhea_chebi/main.nf

```diff
@@ -0,0 +1,22 @@
+process REFORMAT_RHEA_CHEBI {
+    label 'light'
+    container 'quay.io/biocontainers/python:3.10'
+
+    output:
+    path 'rhea_chebi_mapping_*.tsv', emit: tsv_rhea_chebi_mapping
+
+    script:
+    
+    """
+        # Check if params.rhea_chebi_download_mapping is a URL or a file
+    if [[ "${params.rhea_chebi_download_mapping}" =~ ^https?:// ]]; then
+        # If it's a URL, download the file
+        wget ${params.rhea_chebi_download_mapping} -O rhea-reactions.txt.gz
```

Comment:
just want to check, is this a really big file? because input paths can be URLs in nextflow... but obviously if it's too large to download on a per-job basis then something like this does make sense

Reply:
It's a tiny file, unfortunately nextflow fails to download it when I'm providing this url as input path to file() method...
```
WARN: Unable to stage foreign file: https://ftp.expasy.org/databases/rhea/txt/rhea-reactions.txt.gz (try 1) -- Cause: Unable to access path: https://ftp.expasy.org/databases/rhea/txt/rhea-reactions.txt.gz
```

I asked Martin and Vangelis about this and they suggested just using `wget` instead

Reply:
That is so weird, it should work.

This is in the nextflow console:
```
groovy> f = file("https://ftp.expasy.org/databases/rhea/txt/rhea-reactions.txt.gz") 
 
Result: /databases/rhea/txt/rhea-reactions.txt.gz
```

Reply:
🤔 weird
Was it executed on codon? For me, it worked locally but not from codon...

Reply:
Changing `https` to `ftp` helped!